src/translations/en.json

```
@@ -1967,6 +1967,8 @@
           "system_information": "System information",
           "integration_startup_time": "Integration startup time",
           "copy": "Copy",
+          "reported_by": "Reported by",
+          "by": "By",
```

```suggestion
          "reported_by": "Reported by {integration}",
          "by": "By {integration}",
```

Then pass in the domainName as a localization parameter. That's the nicer way for translators to know what they should expect, and you don't need to mess with spaces in the code.